.env.example

```diff
@@ -1,5 +1,3 @@
 URL_AUTH_SERVICE=
-DB_SERVER=
-DB_USER=
-DB_PASSWORD=
 DB_NAME=
+HOST_IP=
```

Comment:
HOST_IP is a mischievous name, as one should normally provide a DNS

Reply:
Would **MONGO_ENDPOINT** more informative ?

Reply:
It is a url, isn’t it? Full with schema, host, port, path?

Reply:
I Agree. MONGO_URL seems ok to me.